### PR TITLE
Story: [CCLS 2282] Notification Attachment Endpoints

### PR DESCRIPTION
**Pull request by farrell-m**

### CODE REVIEW

File: caab-service/src/main/java/uk/gov/laa/ccms/caab/api/service/NotificationAttachmentService.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  /**
+   * Create a new NotificationAttachment entry.
+   *
+   * @param notificationAttachmentDetail - the notification attachment to create.
+   * @return the id of the newly created notification attachment.
+   */
+  public Long createNotificationAttachment(
+      final NotificationAttachmentDetail notificationAttachmentDetail) {
+
+    uk.gov.laa.ccms.caab.api.entity.NotificationAttachment createdNotificationAttachment =
```
Comment (porritta) [inline caab-service/src/main/java/uk/gov/laa/ccms/caab/api/service/NotificationAttachmentService.java:79]: Class is imported, so no need for fully qualified?

**Review by porritta — CHANGES_REQUESTED** [review]
One minor comment

File: caab-service/src/main/java/uk/gov/laa/ccms/caab/api/service/NotificationAttachmentService.java (hunk 1 above)
Comment (farrell-m) [reply caab-service/src/main/java/uk/gov/laa/ccms/caab/api/service/NotificationAttachmentService.java]: Fixed

**Review by porritta — APPROVED** [review]
LGTM

**Pull request merged**